README.md

```diff
@@ -1,9 +1,9 @@
 # celestia-app
 
-[![Go Reference](https://img.shields.io/badge/godoc-reference-blue.svg)](https://pkg.go.dev/github.com/celestiaorg/celestia-app)
+[![Go Reference](https://img.shields.io/badge/godoc-reference-blue.svg)](https://pkg.go.dev/github.com/celestiaorg/celestia-app/v2)
```

Comment:
```suggestion
[![Go Reference](https://img.shields.io/badge/godoc-reference-blue.svg)](https://pkg.go.dev/github.com/celestiaorg/celestia-app)
```

Reply:
I think actually this one we want to keep

Reply:
I think we should just create an issue as a reminder and use the non v2 link for now